offchain/rollups-events/tests/tls.Dockerfile

```
@@ -0,0 +1,65 @@
+# Copyright Cartesi Pte. Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+FROM rust:1.67.0-bullseye AS chef
+
+ENV CARGO_REGISTRIES_CARTESI_INDEX=https://github.com/cartesi/crates-index
+RUN rustup component add rustfmt
+RUN cargo install cargo-chef
+
+FROM chef AS planner
+
+COPY . /usr/src/offchain
+WORKDIR /usr/src/offchain
+RUN cargo chef prepare --bin rollups-events --recipe-path recipe.json
+
+FROM chef AS builder
+
+RUN <<EOF
+DEBIAN_FRONTEND="noninteractive" apt-get update
+DEBIAN_FRONTEND="noninteractive" apt-get install -y --no-install-recommends openssl
+EOF
+
+COPY --from=planner /usr/src/offchain/recipe.json recipe.json
+RUN cargo chef cook --release --recipe-path recipe.json
+
+COPY . /usr/src/offchain
+WORKDIR /usr/src/offchain/rollups-events
+
+RUN cargo build --release --bin broker-tls-test
+
+WORKDIR /usr/src/offchain/rollups-events/tests/certs
+RUN ./gen-test-certs.sh
+
+FROM redis:6.2 as runtime
+
+RUN <<EOF
+DEBIAN_FRONTEND="noninteractive" apt-get update
+DEBIAN_FRONTEND="noninteractive" apt-get install -y --no-install-recommends ca-certificates
+EOF
+
+COPY --from=builder /usr/src/offchain/target/release/broker-tls-test /usr/local/bin
+
+COPY --from=builder /usr/src/offchain/rollups-events/tests/certs/* /data/certs/
+COPY --from=builder /usr/src/offchain/rollups-events/tests/certs/ca.crt /usr/local/share/ca-certificates/
+COPY --from=builder /usr/src/offchain/rollups-events/tests/certs/server.crt /etc/ssl/certs/
+COPY --from=builder /usr/src/offchain/rollups-events/tests/certs/server.key /etc/ssl/private/
+RUN update-ca-certificates
+
+CMD ["redis-server", \
+    "--tls-port 6379", \
+    "--port 0", \
+    "--tls-cert-file /data/certs/server.crt", \
+    "--tls-key-file  /data/certs/server.key", \
+    "--tls-ca-cert-file /data/certs/ca.crt", \
```

Could you do that instead of creating a data dir?
```suggestion
    "--tls-cert-file /etc/ssl/certs/server.crt", \
    "--tls-key-file  /etc/ssl/private/server.key", \
    "--tls-ca-cert-file /usr/local/share/ca-certificates/ca.crt", \
```

---

Here's the thing: `/etc/ssl/private` belongs to root. I've tried modifying the permissions, but to no avail: redis-server always fails to start with a permission error 😕. But the only certificate that needs to be installed is the one from the CA, so I'll remove lines 61 and 62 and use the `/data/certs` in the parameters.